go/saltpack/specs/saltpack_encryption.md

```diff
@@ -160,72 +187,79 @@ recipients, then she will have the key, and she could modify the payload. The
 **authenticators list** is there to prevent this attack, protecting the
 recipients from each other.
 
-We compute each recipient's authenticator in three steps:
+We compute the authenticators in three steps:
```

Comment:
Right above here, you say "protecting the recipient from each other." That used to be true, but we're doing double-duty here and also protecting the header. And this is the only we we protect the header (only the payload which is double-protected).